module_utils/network_lsr/argument_validator.py

```diff
@@ -93,6 +96,10 @@ def get_default_value(self):
             return self.default_value
 
     def validate(self, value):
+        if self.deprecated_by:
+            msg = f"{self.name} is deprecated by {self.deprecated_by}."
```

Comment:
Unfortunately, the module needs to work with Python 2.6, as well. Therefore this syntax does not work. You can probably use:
`"{0.name} is deprecated by {0.deprecated_by}".format(self)`

Reply:
I think this is fixed now too, thanks!